package.json

```diff
@@ -84,7 +85,7 @@
     "yaml-js": "^0.2.0"
   },
   "scripts": {
-    "start": "(cp -n .env.example .env || true) && gulp build && concurrently -r \"gulp watch\" \"webpack-dev-server --port 9090\"",
+    "start": "(cp -n .env.example .env || true) && gulp build && concurrently -r \"gulp watch\" \"webpack-dev-server --https --port 9090\"",
```

Comment:
would you put an information in the readme that it has to use https because of ledger etc.?